src/main/java/org/jenkinsci/plugins/ewm/strategies/MostUsableSpaceStrategy.java

```diff
@@ -38,4 +39,21 @@ public Disk allocateDisk(List<Disk> disks) {
 
         return selectedDisk;
     }
+
+    /**
+     * Calculates the usable space for the given {@link Disk} entry.
+     *
+     * @param disk the disk entry
+     * @return the usable space for the disk
+     * @throws AbortException if mounting point from Master to Disk is {@code null}
+     */
+    private long retrieveUsableSpace(Disk disk) throws AbortException {
```

Comment:
I am not a big fan of AbortException, because it is not extensible in the current version